br/pkg/lightning/backend/external/engine.go

```diff
@@ -293,6 +294,8 @@ func (e *Engine) LoadIngestData(
 	regionRanges []common.Range,
 	outCh chan<- common.DataAndRange,
 ) error {
+	// TODO(lance6716): just try to avoid OOM 😭
```

Comment:
Could we remove this emoj？
```suggestion
	// TODO(lance6716): just try to avoid OOM
```

Reply:
https://github.com/pingcap/tidb/pull/49545/commits/8c2a27c84f26ce9d2b82de5cf4fb88f552dcd3a7

deleted these lines, because they didn't prevent OOM